docs/examples/standard-linter-v2.md

```diff
@@ -0,0 +1,78 @@
+# Standard Linter v2
+
+The [Standard Linter API v2](../types/standard-linter-v2.md) demoed below supports [Messages v2](../types/linter-message-v2.md) only.
+
+## package.json
+
+Only the `providedServices` field is important in this manifest, the rest are just for reference
+
+```json
+{
+  "name": "my-linter",
+  "main": "index.js",
+  "version": "0.0.0",
+  "private": true,
+  "description": "My Linter Package",
+  "keywords": ["linter-example", "example"],
+  "engines": {
+    "atom": ">=1.4.0 <2.0.0"
+  },
+  "providedServices": {
+    "linter": {
+      "versions": {
+        "2.0.0": "provideLinter"
+      }
+    }
+  }
+}
+```
+
+## index.js
+
+```js
+'use babel'
+
+export function activate() {
+  // Fill something here, optional
+}
+export function deactivate() {
+  // Fill something here, optional
+}
+export function provideLinter() {
+  const linterProvider = {
+    name: 'Awesomeness',
```

Comment:
I try to put the name of the base linter program in the `keywords` in package.json as well, which is given as "example" currently. These should be consistent.